nixos/modules/services/misc/bazarr.nix

```diff
@@ -0,0 +1,80 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.bazarr;
+in
+{
+  options = {
+    services.bazarr = {
+      enable = mkEnableOption "bazarr";
+
+      dataDir = mkOption {
+        type = types.str;
+        default = "/var/lib/bazarr/data";
+        description = "The directory where bazarr stores its data files.";
+      };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Open ports in the firewall for the bazarr web interface.";
+      };
+
+      listenPort = mkOption {
```

Comment:
Does bazarr have a "listenAddress"-like option? Does it listen on 0.0.0.0 by default or something?

Reply:
Unfortunately not. Output from `bazarr --help`:

```
usage: bazarr.py [-h] [-c DIR] [-p PORT] [--no-update [BOOL]] [--debug [BOOL]]
                 [--release-update [BOOL]]

optional arguments:
  -h, --help            show this help message and exit
  -c DIR, --config DIR  Directory containing the configuration (default: /nix/
                        store/l1yj07kwvm3g3pyk2mym8zm3vanbm347-bazarr-v0.8.4.3
                        /src/data)
  -p PORT, --port PORT  Port number (default: 6767)
  --no-update [BOOL]    Disable update functionality (default: False)
  --debug [BOOL]        Enable console debugging (default: False)
  --release-update [BOOL]
                        Enable file based updater (default: False)
```